components/template/TemplateCategories.vue

```diff
@@ -53,12 +79,18 @@ export default {
   data() {
     return {
       showCategories: true,
-      isCollapsed: false,
+      isCollapsed: { industry: true }, // Hardcoded industry to open industry collapse by default
+      isCollapsedSm: false,
     }
   },
+  methods: {
+    toggleCollapse(kind) {
```

Comment:
can you use some better naming convention here. Kind sounds much more generic here. You can use categoryType or categoryGroup.